src/pages/PromptPage.vue

```diff
@@ -67,13 +68,14 @@ const prompt = computed(() => {
 onMounted(async () => {
   if (!promptStore.getPrompts.length) {
     await promptStore.fetchPrompts().catch((error) => errorStore.throwError(error))
-    await new Promise((resolve) => setTimeout(resolve, 2000)) // wait 2 seconds before continue
+    await new Promise((resolve) => setTimeout(resolve, 5000)) // wait 2 seconds before continue
   }
 
-  if (!prompt.value) {
-    router.push('/404')
-    return
-  }
+  setTimeout(() => {
+    if (!prompt.value) {
+      router.push('/404')
+    }
+  }, 5000)
```

Comment:
I don't think this is necessary. You can just place the `await` outside the condition.

Reply:
I think this `await new Promise((resolve) => setTimeout(resolve, 5000)) ` is blocking 
And this is async 
```js
setTimeout(() => {
    if (!prompt.value) {
      router.push('/404')
    }
  }, 5000)
```

Reply:
Noted, let's test it in `develop`.